modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro

```diff
@@ -240,6 +240,23 @@ definition {
 		PortletEntry.save();
 	}
 
+	macro addObjectMultiPicklistFieldEntryViaUI {
+		LexiconEntry.gotoAdd();
+
+		Click(locator1 = "ObjectAdmin#ENTRY_CHOOSE_OPTIONS_MULTIPICKLIST");
+
+		MenuItem.clickNoError(menuItem = ${picklistOption});
+
+		if (IsElementPresent(key_text = ${saveName}, locator1 = "Button#ANY")) {
+			Click(
+				key_text = ${saveName},
+				locator1 = "Button#ANY");
+		}
+		else {
+			PortletEntry.save();
```

Comment:
Are these the same thing? Both of them will click save. 
Regardless, these can be removed from the macro. Having the save in the macro step doesn't allow for a lot of future flexibility such as if we have to add from the options and assert an error instead of saving.